lib/widgets/display_movie_detail.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import '/model/movie.dart';
+import '/shared/constants.dart';
+import 'image.dart';
+import '/routes/movie_detail.dart';
+
+class DisplayMovie extends StatelessWidget {
+  const DisplayMovie({super.key, required this.movie});
+
+  final Movie movie;
+  final SizedBox spaceBetweenData = const SizedBox(
+    height: 10,
+  );
+  final TextStyle movieTitleTextStyle = const TextStyle(
+    fontSize: 24,
+    fontFamily: 'Roboto',
```

Comment:
I see that you are using the Roboto theme for almost all the TextStyle, you can set this theme at the route of the App

Reply:
solved